test/e2e/alertmanager_test.go

```diff
@@ -28,86 +28,89 @@ import (
 )
 
 func TestAlertmanagerCreateDeleteCluster(t *testing.T) {
+	t.Parallel()
+
+	ctx := testFramework.NewTestCtx(t)
```

Comment:
@fabxc Ok, I have changed this. Do you want to save the kubeClient in each ctx on creation as well? Would seem strange to me.